src/status_im/contexts/communities/events_test.cljs

```diff
@@ -181,9 +181,4 @@
 
     (testing "given a community with lower clock"
       (let [effects (events/handle-community {:db {:communities {community-id {:clock 3}}}} [community])]
-        (is (nil? effects))))
-
-    (testing "given a community without clock"
-      (let [community #js {:id community-id}
-            effects   (events/handle-community {} [community])]
```

Comment:
The test is incorrect. Instead of evaluating the community without a clock, it is currently testing the community with a clock but an empty eb.